### PR TITLE
Update Marathon (Engineering revamp and small stuff)

### LINKED ISSUE #29361: Marathon AME room unused APC
image

This AME is unconnected to MV wires. The LV networks connects through a wall or two to another APC that covers a wider area of engineering so it causes no power issues however.

### LINKED ISSUE #29570: The section of Marathons maints under chem is unpowered
## Description
Title. Two doors do not receive power for some reason.

**Reproduction**
Step 1. Load Marathon Station
Step 2. Check maints (under/around chem)
Step 3. Slam face first into a door

**Screenshots**

**Additional context**
The station was completely default, no wiring changes. Power is NOT a concern, as the SMES had the StationInfiniteBattery components.

### LINKED ISSUE #29776: Marathon engineering is horrible
## Description
I'm going to refer to BOTH the singulo/tesla as the singulo because I'm on a phone really mad at marathon's design

TL;DR: Marathon has constant power issues because (a.) it doesn't have it's singulo encasing pre-built and (b.) doesn't have enough power round start to run the station before getting singulo set up which causes head splitting headaches to any poor engineer(s) that even dare try and play on this map.

For a map that is supposed to be powered by the singulo it is a pain to do so. Building the singulo with parts can take a good 15-30 minutes and *sourcing* the parts is an issue in on itself because unless cargo has the mats for it the materials that come scattered across engineering are usually never enough, not to mention engineering has to literally build the entire singulo by themselves with no outer protection towards asteroids. I don't even think you can set up the singulo without buying AME refuels from cargo. When I last played on Marathon it took a good hour for engineering to get everything done (by the way: marathon only supports _only four engineers_ minus CE!) and by the time that singulo was set up, evac had already arrived.

BLUF

### PR DESCRIPTION
**Pull request by lzk228**
marathon CL

- grill in kitchen
- revamp hydro, add animal zone
- add more south shielding for northwest solars, add more connections with the station
- make sec meteor shielding more sane
- add frequently used electronics to electronics vault
- shutters in dorms
- replace one arti in sci with container (there is another arti in cargo, so it's 3 artifacts at all on the station + 1 in secret stash south of bridge)
- revamp disposals
- revamp rnd entrance
- add small lights to hallway to chapelroid
- library and little power revamp on chapelroid
- fixed missing lv cable in chem maints (fixes #29570)
- add one entrance from hallway to canister storage
- remove unused mv in chem maints
- revamp atmos locker room

and the biggest one
- revamp engineering department (atmos not included) (fixes #29776 fixes #29361)

# Old
Old-Marathon-0

# New
Below from bot

### CODE REVIEW

**Comment by Plykiya** [comment]
as a librarian main it would be nice if the librarian bedroom was expanded down to be one tile bigger to at least fit a dresser, or to have a door to the game room :^)

**Comment by lzk228** [comment]
> as a librarian main it would be nice if the librarian bedroom was expanded down to be one tile bigger to at least fit a dresser, or to have a door to the game room :^)

use crowbar, wrench and wood to replace the table with a dresser
